algebra/ring.lean

```diff
@@ -57,6 +57,100 @@ section comm_ring
   ⟨dvd_of_neg_dvd, neg_dvd_of_dvd⟩
 end comm_ring
 
+def is_unit {α : Type u} [comm_ring α] (x : α) := ∃ y, x * y = 1
```

Comment:
what is the difference between this `is_unit` and `units` in `algebra/group`?
the best would be to split `units` into a predicate on `val`, and then have `units` as a subype.